sound/soc/sof/ipc.c

```diff
@@ -654,6 +664,133 @@ static void sof_set_get_mmap(struct snd_sof_dev *sdev,
 				       cdata->chanv, send_bytes);
 }
 
+static int sof_get_ctrl_size_params(struct snd_sof_control *scontrol,
+				    struct sof_ipc_ctrl_data *cdata,
+				    struct sof_ctrl_data_send_params *sipc)
```

Comment:
sparams?
Not sure what sipc is supposed to hint at?

Reply:
ok